runtime-light/stdlib/file/php-streams/php-streams.cpp

```diff
@@ -0,0 +1,30 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "runtime-light/stdlib/file/php-streams/php-streams.h"
+
+#include <cstring>
+
+#include "runtime-light/state/instance-state.h"
+
+namespace {
+
+constexpr std::string_view STDERR_NAME{"stderr"};
```

Comment:
Let's initialise globals with `=`. It's just how we started doing it... Also, as I can see, you also use it in another file.